### ISSUE TITLE
The Tricorder is borked

### ISSUE THREAD

**Issue by LordWalnut** [DDMers/StarTrek13#57]
[Round ID]: Test Server

[Reproduction]: Once the menu for the tricorder opens, it refuses to go away, and to make matters worse, none of the options actually do anything, once you click an option, the menu just pops back up. The only way to remove the menu is to either disconnect or throw the tricorder away from you.

**Comment by Cdey78** [comment]
Yeah, has to do with my shitcode and Kmc somehow adapting that shitcode.
You can somehow close the menu by doing something weird with the popup, I forgot how though.

**Comment by LordWalnut** [comment]
Found out that it's double-clicking that refuses to work, you have to select the option and hit 'OK' or it will just open again and again.

**Comment by Cdey78** [comment]
KMC improved the tricorder code, except for the menu bullshit. Redundant issue.

**Comment by limofs** [comment]
[Round ID]: 261 
[Reproduction]: When hiding tricorder scanner in a bag you cant remove it from there.

**Comment by Drobe50** [comment]
This issue is still happening, even with borg buildmode.
